src/Operators/finitedifference.jl

```diff
@@ -937,28 +937,32 @@ return_eltype(::AdvectionOperator, velocity, arg) = eltype(arg)
     U = UpwindBiasedProductC2F(;boundaries)
     U.(v, x)
 
-Compute the product of a face-valued vector field `v` and a center-valued field
-`x` at cell faces by upwinding `x` according to `v`
+Compute the product of the face-valued vector field `v` and a center-valued
+field `x` at cell faces by upwinding `x` according to the direction of `v`.
+
+More precisely, it is computed based on the sign of the 3rd contravariant
+component, and it returns a `Contravariant3Vector`:
 ```math
-U(v,x)[i] = \\begin{cases}
-  v[i] x[i-\\tfrac{1}{2}] \\textrm{, if } v[i] > 0 \\\\
-  v[i] x[i+\\tfrac{1}{2}] \\textrm{, if } v[i] < 0
+U(\\boldsymbol{v},x)[i] = \\begin{cases}
+  v^3[i] x[i-\\tfrac{1}{2}]\\boldsymbol{e}_3 \\textrm{, if } v^3[i] > 0 \\\\
+  v^3[i] x[i+\\tfrac{1}{2}]\\boldsymbol{e}_3 \\textrm{, if } v^3[i] < 0
   \\end{cases}
 ```
+where ``\\boldsymbol{e}_3`` is the 3rd covariant basis vector.
 
 Supported boundary conditions are:
-- [`SetValue(x₀)`](@ref): set the value of `x` to be `x₀` on the boundary. On
+- [`SetValue(x₀)`](@ref): set the value of `x` to be `x₀` on the opposing side of the boundary. On
```

Comment:
What does "opposing side of the boundary" means here? It would sound like it would be the opposing face at index 3/2, but it shouldn't, as  `x₀` lives at cell center.

Reply:
I clarified that this means a ghost cell.